### ISSUE TITLE
Bug when a bot has no real name

### ISSUE THREAD

**Issue by cvergne** [clem/slack-to-wallabag#12]
Hi,

When running `php bin/console stw:crawl:slack`, I got the following error : 
```
In AbstractMySQLDriver.php line 118:        
                                                                                                                                                            
  An exception occurred while executing 'INSERT INTO slack_user (slack_id, username, real_name, avatar) VALUES (?, ?, ?, ?)' with params ["XXXXXXXXX", "nom-bot", null, null]:  
                                                                                                                                                                       
  SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'real_name' cannot be null                                                                                       
                                                                                                                                                                                

In PDOStatement.php line 107:                                                                               
  SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'real_name' cannot be null  
                                                                                           

In PDOStatement.php line 105:
                                                                                           
  SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'real_name' cannot be null  
```

_(Note: I replaced the "slack_id" in the error message)_

Maybe it would be good to fill the `real_name` *(if empty)* with the `username` and keep the `real_name` **not** nullable.

**Comment by clem** [comment]
Hi,

Thanks for the bug report. 
A merge request has been made (#13) and will be merged ASAP.

**Comment by clem** [comment]
Hi,

This issue has been fixed in the #13 :)
